packages/core/src/clone/CloneManager.ts

```
@@ -103,98 +103,90 @@ export class CloneManager {
+    // @shallowClone / @deepClone: deep copy complex objects
+    const type = sourceProperty.constructor;
+    switch (type) {
+      case Uint8Array:
+      case Uint16Array:
+      case Uint32Array:
+      case Int8Array:
+      case Int16Array:
+      case Int32Array:
+      case Float32Array:
+      case Float64Array:
+        let targetPropertyT = <TypedArray>target[k];
```

Biome (2.4.7)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)